styles/variables-base.less

```diff
@@ -322,8 +322,8 @@
 // ---------------------------------------
 @agate-indicator-align-self:    inherit;
 @agate-indicator-border-radius: inherit;
-@agate-indicator-display:       inherit;
-@agate-indicator-height:        inherit;
+@agate-indicator-display:       0;
+@agate-indicator-height:        0;
```

Comment:
these 2 variables look to not be used in any component style. Can you check and remove them for all skins if necesary?

Reply:
They are used in Silicon skin to render the dot in selected state. They are not needed in other skins, so that's why I set them to `0`.